packages/components/src/internal/app/constants.ts

```diff
@@ -20,7 +20,7 @@ export const LABKEY_SERVER_PRODUCT_NAME = 'LabKey Server';
 const FREEZER_MANAGER_PRODUCT_NAME = 'Freezer Manager';
 
 const BIOLOGICS_CONTROLLER_NAME = 'biologics';
-const SAMPLE_MANAGER_CONTROLLER_NAME = 'sampleManager';
+export const SAMPLE_MANAGER_CONTROLLER_NAME = 'sampleManager';
```

Comment:
For consistency, should use SAMPLE_MANAGER_APP_PROPERTIES.controllerName instead of export this